### PR TITLE
docs: update adev conditionals-and-loops.md

### PR DESCRIPTION
**Pull request by prasantht96**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: adev/src/content/introduction/essentials/conditionals-and-loops.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class UserControls {
 }
 ```
 
-In this example, Angular only renders the `<section>` element if the `isAdmin` property is true. Otherwise, it does not appear page.
+In this example, Angular only renders the `<section>` element if the `isAdmin` property is true. Otherwise, it does not appear on the page.
```
Comment (SvetlaGeorgieva) [inline adev/src/content/introduction/essentials/conditionals-and-loops.md:35]: Also, instead of "`<section>` element" should be "`<button>` element"

**Review by twerske — APPROVED** [review]

File: adev/src/content/introduction/essentials/conditionals-and-loops.md (hunk 1 above)
Comment (twerske) [reply adev/src/content/introduction/essentials/conditionals-and-loops.md:35]: This is fixed in another PR I've reviewed!

**Comment by twerske** [comment]
Please fix merge conflicts!

**Comment by prasantht96** [comment]
> Please fix merge conflicts!

Merge Conflicts Fixed.

**Review by SvetlaGeorgieva — APPROVED** [review]

**Comment by twerske** [comment]
You'll need to squash commits here still!

**Pull request closed without merging**